ports/unimail-cpp-sdk/portfile.cmake

```
@@ -0,0 +1,24 @@
+set(VCPKG_BUILD_TYPE release)
+set(VCPKG_POLICY_MISMATCHED_NUMBER_OF_BINARIES enabled)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO unimails/unimail-cpp-sdk
+    REF "v${VERSION}"
+    SHA512 96587849338e7f88a322ddb937a43aa245026157254d22e2e186ad8b6f6a7f655c15943573e5d87c991e627d9a7b419e931f24694d0579973f2dae81ffd73d87
+    HEAD_REF master
+)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS "-DUNIMAIL_TEST=OFF"
```

Prepare for future additions.
```suggestion
    OPTIONS
        -DUNIMAIL_TEST=OFF
```